index.html

```diff
@@ -334,7 +380,43 @@ <h2 class="contact-title">Contact me</h2>
         </div>
       </section>
     </section>
-
+    <div class="popup-window" id="popup">
+      <section class="popup">
+        <div class="popup-header">
+          <h2 class="popup-project-title" id="popup-project-title"></h2>
+          <span class="close">&times;</span>
+        </div>
+        <div class="popup-project-info" id="popup-project-info"></div>
+        <div id="popup-project-image-holder">
+          <img
+            src="#"
+            alt="Tonic project screenshot"
+            class="popup-project-image"
+            id="popup-project-image"
+          />
+        </div>
+        <div class="popup-project-details">
+          <div class="column-left">
+            <p
+              class="popup-project-description"
+              id="popup-project-description"
+            ></p>
+          </div>
+          <div class="column-right">
+            <div class="popup-project-stack" id="popup-project-stack"></div>
+            <div class="popup-project-buttons">
+              <button type="button" id="see-live-btn" onclick="">
+                See Live
+              </button>
+              <button type="button" id="see-source-btn" onclick="">
+                See Source
+              </button>
+            </div>
```

Comment:
Maybe we could do the same as for the header for the footer. I can see the buttons are missing the icons and are not taking the full width of the card in the mobile version.

Yours:
![image](https://user-images.githubusercontent.com/79114233/141318373-959b335b-8b0f-440a-a7e4-551b79e47f29.png)


Template:
![image](https://user-images.githubusercontent.com/79114233/141318314-13c2783e-82fb-4892-bade-e1bf07e094d9.png)